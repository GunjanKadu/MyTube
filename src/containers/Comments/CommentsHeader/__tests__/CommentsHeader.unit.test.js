import React from 'react';
import {shallow} from 'enzyme';
import {CommentsHeader} from '../CommentsHeader';

test('CommentsHeader renders with props.amountComments = null', () => {
  const wrapper = shallow(
    <CommentsHeader/>
  );
  expect(wrapper).toMatchSnapshot();
});

test('CommentsHeader renders with props.amountComments = 0', () => {
  const wrapper = shallow(
    <CommentsHeader amountComments={0}/>
  );
  expect(wrapper).toMatchSnapshot();
});

test('CommentsHeader renders with props.amountComments = 1', () => {
  const wrapper = shallow(
    <CommentsHeader amountComments={1}/>
  );
  expect(wrapper).toMatchSnapshot();
});

test('CommentsHeader renders with props.amountComments = 5', () => {
  const wrapper = shallow(
    <CommentsHeader amountComments={5}/>
  );
  expect(wrapper).toMatchSnapshot();
});