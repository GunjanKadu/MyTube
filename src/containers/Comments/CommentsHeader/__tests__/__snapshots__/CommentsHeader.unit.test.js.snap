// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader renders with props.amountComments = 0 1`] = `
<div
  className="comments-header"
>
  <h4>
    0
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
    role="button"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;

exports[`CommentsHeader renders with props.amountComments = 1 1`] = `
<div
  className="comments-header"
>
  <h4>
    1
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
    role="button"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;

exports[`CommentsHeader renders with props.amountComments = 5 1`] = `
<div
  className="comments-header"
>
  <h4>
    5
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
    role="button"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;

exports[`CommentsHeader renders with props.amountComments = null 1`] = `
<div
  className="comments-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
    role="button"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;
